src/components/layout/sendTip/SendTip.vue

```diff
@@ -115,31 +115,33 @@ export default {
       const amount = shiftDecimalPlaces(this.sendTipForm.amount,
         this.inputToken !== null ? this.tokenInfo[this.inputToken].decimals : 18).toFixed();
 
-      this.$store.dispatch('aeternity/tip', {
-        url: this.sendTipForm.url,
-        title: this.sendTipForm.title,
-        amount,
-        tokenAddress: this.inputToken,
-      }).then(async () => {
+      try {
+        await this.$store.dispatch('aeternity/tip', {
+          url: this.sendTipForm.url,
+          title: this.sendTipForm.title,
+          amount,
+          tokenAddress: this.inputToken,
+        });
         this.clearTipForm();
         this.$store.dispatch('modals/open', {
           name: 'success',
           title: this.$t('components.layout.SendTip.SuccessHeader'),
           body: this.$t('components.layout.SendTip.SuccessText'),
         });
         EventBus.$emit('reloadData');
-      }).catch((e) => {
-        this.sendingTip = false;
-        if (e.code && e.code === 4) {
+      } catch (error) {
+        if (error.code && error.code === 4) {
           return;
```

Comment:
why return without anything?

Reply:
The author wanted to omit this kind of exceptions. It was introduced in https://github.com/aeternity/superhero-ui/commit/0b5506d5e7e24bef9376b15e425cc2cf6ce7ecfd I can't get what is supposed to fix, proposing to remove

Reply:
The author wanted to omit this kind of exception. It was introduced in https://github.com/aeternity/superhero-ui/commit/0b5506d5e7e24bef9376b15e425cc2cf6ce7ecfd I can't get what is supposed to fix, proposing to remove

Reply:
ok :D good question